.github/actions/spelling/patterns.txt

```diff
@@ -1,21 +1,9 @@
-[Ll][Oo][Mm][Bb][Ii][Qq]
 (\((.*?)\))
```

Comment:
@Luko6 what's this pattern for?

Reply:
This will skip anything in parentheses, see https://regexr.com/73tll for an explanation.

Seems problematic to me.

Reply:
@Luko6 before we remove this, what was the intention here so that we can find a proper replacement?

Reply:
I think this is used to skip the links inside parentheses in markdown files

Reply:
OK, so the problem what that is that it's too broad, like Oliver explained. Weren't the URL patterns sufficient for this use-case?